CMakeLists.txt

```diff
@@ -0,0 +1,62 @@
+cmake_minimum_required(VERSION 3.1)
+project(fastor VERSION 0.6.4)
```

Comment:
This version number will have to be updated after cutting a release.

Reply:
I agree.